ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,177 @@
+    name: knative-eventing-src
+    namespace: openshift
+    tag: knative-v1.8
+  openshift_knative-v1.1.0_knative-eventing-kafka-src:
+    name: knative-v1.1.0
+    namespace: openshift
+    tag: knative-eventing-kafka-src
+  openshift_knative-v1.8.0_knative-serving-src:
+    name: knative-v1.8.0
+    namespace: openshift
+    tag: knative-serving-src
+  openshift_release_golang-1.18:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+images:
+- context_dir: openshift/ci-operator/source-image
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    openshift_eventing-kafka-broker-src_knative-v1.7:
+      as:
+      - registry.ci.openshift.org/openshift/eventing-kafka-broker-src:knative-v1.7
+    openshift_knative-eventing-src_knative-v1.8:
+      as:
+      - registry.ci.openshift.org/openshift/knative-eventing-src:knative-v1.8
+    openshift_knative-v1.1.0_knative-eventing-kafka-src:
+      as:
+      - registry.ci.openshift.org/openshift/knative-v1.1.0:knative-eventing-kafka-src
+    openshift_knative-v1.8.0_knative-serving-src:
+      as:
+      - registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-src
+  to: serverless-operator-src
+- context_dir: .
+  dockerfile_path: serving/ingress/Dockerfile
+  from: base
+  inputs:
+    openshift_release_golang-1.18:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.18
+  to: knative-openshift-ingress
+- context_dir: .
+  dockerfile_path: knative-operator/Dockerfile
+  from: base
+  inputs:
+    openshift_release_golang-1.18:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.18
+  to: knative-operator
+- context_dir: .
+  dockerfile_path: openshift-knative-operator/Dockerfile
+  from: base
+  inputs:
+    openshift_release_golang-1.18:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.18
+  to: openshift-knative-operator
+- context_dir: .
+  dockerfile_path: serving/metadata-webhook/Dockerfile
+  from: base
+  inputs:
+    openshift_release_golang-1.18:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.18
+  to: metadata-webhook
+- context_dir: olm-catalog/serverless-operator
+  from: base
+  to: serverless-bundle
+- context_dir: olm-catalog/serverless-operator
+  dockerfile_path: stopbundle.Dockerfile
+  from: base
+  to: serverless-stop-bundle
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: operator-e2e-interop-aws-ocp413
+  cron: 0 1 * * 1
+  steps:
+    allow_best_effort_post_steps: true
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: aws.interop.ccitredhat.com
+    post:
```

Comment:
Since we are explicitly using post here in a test step it is overriding the post steps in the `ipi-aws` workflow causing the cluster deprovision steps not to run.

Looking into solution now and I will manually delete the 3 clusters that we left up. Please don't run another job until this is updated.